src/pages/errorpages/Forbidden.tsx

```diff
@@ -1,20 +1,62 @@
-import { Box, Link as MuiLink, Typography } from '@mui/material';
-import { useTranslation } from 'react-i18next';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import LockOutlineIcon from '@mui/icons-material/LockOutline';
+import LoginIcon from '@mui/icons-material/Login';
+import { Box, Button, SxProps, Typography } from '@mui/material';
+import { Trans, useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
 import { Link } from 'react-router';
+import { RootState } from '../../redux/store';
+import { dataTestId } from '../../utils/dataTestIds';
+import { useAuthentication } from '../../utils/hooks/useAuthentication';
 import { UrlPathTemplate } from '../../utils/urlPaths';
 
+const buttonProps: SxProps = { textTransform: 'none', mt: '1rem', width: 'fit-content' };
```

Comment:
Hadde kanskje vært mer naturlig å ha dette som en `StyledButton`, og brukt den i stedet for å legge til `sx` i etterkant